j2ee.core.utilities/src/org/netbeans/modules/j2ee/core/support/java/method/ExceptionsPanel.form

```diff
@@ -1,26 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 
-<!--
```

Comment:
This should be reverted.

Reply:
Thanks @emilianbold, can I ask why to revert the .form changes?  I took the license headers out because these are auto-generated and I mistakingly put them in the first time.

Reply:
In the mean time the decision was to put them in. And they were added all over the codebase automatically, via a script.

So now your commit appears to remove what was just added.

Reply:
I'm sorry about that.  I am thinking that I will close this pull request and start fresh...sound ok?  Thanks

Reply:
Don't. Just start in another branch on your local machine then push that into your master branch on github (eventually with --force). The existing PR will pick the updated changesets and we can go from there.